core/src/install.js

```diff
@@ -134,4 +136,7 @@ window.addEventListener('DOMContentLoaded', function() {
 		drawTitles: true,
 		nonce: btoa(getToken()),
 	})
+
+	$("#dbpass").showPassword().keyup()
+	$('#adminpass').showPassword().keyup()
```

Comment:
```suggestion
	$('#dbpass').showPassword().keyup()
	$('#adminpass').showPassword().keyup()
```

Weird, it doesn't log the lint error inline.
That is why it fails

Reply:
Oops